### DISCUSSION TITLE
Talks about UI/UX

### DISCUSSION THREAD

**Discussion by SreejithNS** [adrig-technologies/website#7] [Ideas]
All discussion and decisions about UI/UX goes here.

**Comment by ShreyStar02** [comment]
Suggestions for UI:
- Reference Websites - for UI/UX
- https://gsap.com/
- https://www.clubhouse.com/
- https://linktr.ee/
- https://www.realworld.co/
- https://practipago.pe/
- https://keynest.com/
- https://react.dev/
- https://devdreaming.com/videos/create-nft-collection-website-reactjs#overview
- https://priyansh42.github.io/IT-Company-Website/

**Reply by SreejithNS** [reply]
Adding one more ref website with good color gradients usage

https://ine.com

**Comment by SreejithNS** [comment]
**Basic Wireframe**
By @Srinidhi-Arunkumar 

https://www.figma.com/file/mjDN8Y5KLg5uV3KMFLBriZ/Adrig-Website-Wireframes?type=design&node-id=0%3A1&mode=design&t=HsVNJ0g88XW5bJyi-1

**Comment by SreejithNS** [comment]
@Srinidhi-Arunkumar Please add footer, and sections into your wireframe. We will have a meeting if required for discussion or else a conversation here will also work for me.

**Comment by Srinidhi-Arunkumar** [comment]
Sure I will look into it.

On Mon, Nov 20, 2023, 12:08 PM Sreejith N Subramanian <
***@***.***> wrote:

> @Srinidhi-Arunkumar <https://github.com/Srinidhi-Arunkumar> Please add
> footer, and sections into your wireframe. We will have a meeting if
> required for discussion or else a conversation here will also work for me.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/adrig-technologies/website/discussions/7#discussioncomment-7615876>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BAO6NXQ76RJP23V3S2POLJTYFL3FJAVCNFSM6AAAAAA7Q3D7TOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TMMJVHA3TM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>

-- 
**Disclaimer:*
This message was sent from Vellore Institute of Technology.  
The contents of this email may contain legally protected confidential or 
privileged information of “Vellore Institute of Technology”.  If you are 
not the intended recipient, you should not disseminate, distribute or copy 
this e-mail. Please notify the sender immediately and destroy all copies of 
this message and any attachments. If you have received this email in error, 
please promptly notify the sender by reply email and delete the original 
email and any backup copies without reading them.*

**Comment by SreejithNS** [comment]
( Draft ) ( From Jerome )

Website content 
       - About
            core Team members photo 
       
       - what is adrig
            Brief description about us and about us   
       
       - why choose us
            Providing client about necessary details like why they have to choose us 
                                eg.🌟 Proven Track Record: 
                                   🤝 Client-Centric Approach:
                                   🔒 Reliability:
           *add relevant images on  each point(testimonials)                    
                   
       - our services 
            *should present all the services we do with an animation for each ot it 
                 
       - achivements 
               our part records winning the hackathons and other stuff 
               add images like trophy r group pic 

       - our products
               we should put all our products 
                         -SKPC face recoginition 
                         -EyeQ
                         -Zen interactive(sumdomain studio we release hypercasual video games)
                                     *add game videos are any animations relevant to it 


Adding images relevant to it.
(This is not the order of content)

**Comment by Srinidhi-Arunkumar** [comment]
Okay got it.
Will work on it tmrw !

On Mon, Nov 20, 2023, 1:04 AM Sreejith N Subramanian <
***@***.***> wrote:

> *Basic Wireframe*
> By @Srinidhi-Arunkumar <https://github.com/Srinidhi-Arunkumar>
>
>
> https://www.figma.com/file/mjDN8Y5KLg5uV3KMFLBriZ/Adrig-Website-Wireframes?type=design&node-id=0%3A1&mode=design&t=HsVNJ0g88XW5bJyi-1
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/adrig-technologies/website/discussions/7#discussioncomment-7613064>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BAO6NXQXDZARBZ23AY6QNG3YFJNMLAVCNFSM6AAAAAA7Q3D7TOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TMMJTGA3DI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>

-- 
**Disclaimer:*
This message was sent from Vellore Institute of Technology.  
The contents of this email may contain legally protected confidential or 
privileged information of “Vellore Institute of Technology”.  If you are 
not the intended recipient, you should not disseminate, distribute or copy 
this e-mail. Please notify the sender immediately and destroy all copies of 
this message and any attachments. If you have received this email in error, 
please promptly notify the sender by reply email and delete the original 
email and any backup copies without reading them.*

**Comment by SreejithNS** [comment]
(From Jerome) (Content Regarding Services)

Artificial Intelligence (AI):
           Intelligence Redefined
                         Harness the power of AI for unparalleled automation and decision-making. Our solutions optimize processes and drive innovation

Internet of Things (IoT):
           Connecting the Unconnected
                         Seamlessly link devices, unlock data insights, and enhance efficiency with our IoT solutions. Experience the synergy of connected ecosystems.

Software development(web and app):
           Transform your ideas into robust websites and apps. Our development expertise ensures scalable and efficient applications.    

Play Beyond Boundaries:
           Creating games for pure enjoyment on both PC and Mobile platforms. 

Virtual Reality (VR):
           Immerse in Limitless Realities
                         Elevate user experiences through cutting-edge VR solutions. From simulations to interactive environments, we redefine the boundaries of virtual immersion.
